### PR TITLE
Kia: add missing Optima EUR FW versions

### PR DESCRIPTION
**Pull request by eivegau**

### CODE REVIEW

**Comment by sshane** [comment]
What is your dongle ID? For what model year?

**Comment by eivegau** [comment]
*dongle id:* aeba7953fb011c0e
<https://useradmin.comma.ai/?onebox=aeba7953fb011c0e>

2024-07-20, št, 00:18 Shane Smiskol ***@***.***> rašė:

**Comment by sshane** [comment]
|Dongle, platform, VIN|Name from VIN 🆚 CarDocs|Segments|Bad seg tags|Good seg tags|
|:-------------------------------------------------------------------------------------------------------------------------|:--------------------------------|:------------------------------------------|:---------------|:----------------------------------------------------------------------------------|
|[aeba7953fb011c0e](https://useradmin.comma.ai/?onebox=aeba7953fb011c0e)<br><sub>KIA_OPTIMA_G4<br>KNAGW815G........</sub>|KIA  1987 🆚<br>Kia Optima 2017|22 routes,<br>119 segments<br>(2.0 hours)||- valid torque params: 114<br>- all lat active: 4<br>- no input all lat active: 2|

File: selfdrive/car/hyundai/fingerprints.py [hunk 1]
```diff
@@ -760,10 +760,12 @@
       b'\xf1\x00JF__ SCC F-CUP      1.00 1.00 96400-D4100         ',
     ],
     (Ecu.abs, 0x7d1, None): [
-      b'\xf1\x00JF ESC \x0f 16 \x16\x06\x17 58920-D5080',
+      b'\xf1\x00JF ESC \x0f 16 \x16\x06\x17 58920-D5080           ',
+      b'\xf1\x00JF ESC \t 17 \x16\x06# 58920-D4180                ',
```
Comment (sshane) [inline selfdrive/car/hyundai/fingerprints.py:764]:
```suggestion
      b'\xf1\x00JF ESC \x0f 16 \x16\x06\x17 58920-D5080',
      b'\xf1\x00JF ESC \t 17 \x16\x06# 58920-D4180',
```

File: selfdrive/car/hyundai/fingerprints.py [hunk 2]
```diff
@@ -760,10 +760,12 @@
...
     ],
     (Ecu.fwdCamera, 0x7c4, None): [
-      b'\xf1\x00JFWGN LDWS AT USA LHD 1.00 1.02 95895-D4100 G21',
+      b'\xf1\x00JFWGN LDWS AT USA LHD 1.00 1.02 95895-D4100 G21   ',
+      b'\xf1\x00JF__ SCC F-CUP      1.00 1.00 96400-D4100         ',
```
Comment (sshane) [inline selfdrive/car/hyundai/fingerprints.py:768]:
```suggestion
      b'\xf1\x00JFWGN LDWS AT USA LHD 1.00 1.02 95895-D4100 G21   ',
      b'\xf1\x00JF__ SCC F-CUP      1.00 1.00 96400-D4100         ',
```

**Pull request merged**